django/core/handlers/base.py

```diff
@@ -170,11 +167,30 @@ def _get_response(self, request):
             resolver = get_resolver()
 
         resolver_match = resolver.resolve(request.path_info)
```

Comment:
Maybe also assign to request.resolver_match in the same line (if it works with the line limit)